src/_P021_Level.ino

```diff
@@ -18,24 +22,119 @@
 # include "src/Globals/RulesCalculate.h"
 # include "src/WebServer/ESPEasy_WebServer.h"
 
+# define PLUGIN_021_DEBUG
```

Comment:
The directive causes `LOG_LEVEL_DEBUG` messages, but that level is not defined for all build types. You also have to check for `BUILD_NO_DEBUG` and if not defined, you shouldn't enable `PLUGIN_021_DEBUG`.